components/dropdown/dropdown-content-mixin.js

```diff
@@ -569,7 +569,8 @@ export const DropdownContentMixin = superclass => class extends LocalizeCoreElem
 			const activeElement = getComposedActiveElement();
 
 			if (isComposedAncestor(this, activeElement)
-				|| isComposedAncestor(this.__getOpener(), activeElement)) {
+				|| isComposedAncestor(this.__getOpener(), activeElement)
+				|| activeElement === this.__previousFocusableAncestor) {
```

Comment:
I think we'd need this in `PopoverMixin` as well - [here](https://github.com/BrightspaceUI/core/blob/main/components/popover/popover-mixin.js#L957). While the positioning code has been mostly a direct port, we should test this. The `?popover=true` can be used on the dropdown demo pages to try it out.

If in doubt, look for the `#top-layer` at the bottom of the document inspector to confirm it is being used.

Reply:
Unfortunately we'll be dealing with this duplication for a little while. Hopefully dropdown changes will be far & few between.